recipes/shorah/1.99.0/meta.yaml

```diff
@@ -32,8 +33,6 @@ requirements:
     - gsl
     - zlib
     - blas {{ blas }}
-    - setuptools
-    - pip
```

Comment:
Saddly, this [breaks the build on mac OS X](https://circleci.com/gh/bioconda/bioconda-recipes/101896) for some reason (badly written setup.py ?).

I'll try to get our devs to fix that, and will get back to you.